data/map.txt

```diff
@@ -26335,6 +30202,20 @@ planet "Ashy Reach"
 	outfitter "Coalition Basics"
 	"required reputation" 15
 
+planet Atch-werimoural-tinikt
+	attributes farming xapleaux
+	landscape land/fields3
+	description `This world is possibly the most "normal" habitable world by galactic standards found anywhere in the core. With only a few small but extremely deep oceans and located in a darker, cooler region of the core, it almost has a weather cycle that could be termed familiar to outsiders. If Dygit-jarobouillier-cehicot feeds the Home Stretch, Atch-werimoural-tinikt feeds the the Waning Reach and the Quiet Arm, with the majority of its land area given over to agriculture or ecological products used to maintain the terraformed environments of some other Xapleaux worlds.`
+	spaceport "The spaceport is a massive sprawling affair but is largely populated by plants, boxes and machines. The actual number of Xapleaux on this world is relatively low and there's no tourism to speak of, leading to a very business-like atmosphere with a high degree of automation."
+	government Xapleaux
+
+planet Atke-babounriuer-taxerik
+	attributes urban xapleaux
+	landscape land/mountain14-sfiera
+	description "This world was once struck by an object of enormous mass and force, enough to cause the entire crust of the world to ripple violently. Only one relatively flat area of land exists on the surface, a crater surrounded on all sides by concentric rings of mountains which gradually decrease in height as they move away from the crater. Directly opposite the crater, on the other side of the world, a solid circle of mountains form a thicket of jagged peaks that reach almost to space in their center."
+	spaceport "The spaceport, like almost all structures on this world, is built on the side of a mountain at the altitude most compatible with the majority of Xapleaux's needs. Despite the engineering challenges, the Xapleaux's conventional architecture remains at large, with buildings comprised of wide, flat floors layered atop each other in the usual, slightly uneven, pancake-stack style. Only here, these structures jut out over the valleys below like polypores (bracket or shelf fungi), sometimes far enough to give the illusion of floating above a cloudbank when the frequent dense fog fills the valley below."
```

Comment:
`polypores (bracket or shelf fungi` sounds kind of clunky, so I replaced it with something that more players will probably be familiar with. However, I don't know if it still fits, so feel free to ignore this suggestion. 
```suggestion
	spaceport "The spaceport, like almost all structures on this world, is built on the side of a mountain at the altitude most compatible with the majority of the Xapleaux's needs. Despite the engineering challenges, the Xapleaux's conventional architecture remains at large, with buildings comprised of wide, flat floors layered atop each other in the usual, slightly uneven, pancake-stack style. Only here, these structures jut out over the valleys below like cliffs, sometimes far enough to give the illusion of floating above a cloudbank when the frequent dense fog fills the valley below."
```

Reply:
Took the typo, but I think the other change actually loses the meaning. There isn't really anything else out there that fits.
```suggestion
	spaceport "The spaceport, like almost all structures on this world, is built on the side of a mountain at the altitude most compatible with the majority of the Xapleaux's needs. Despite the engineering challenges, the Xapleaux's conventional architecture remains at large, with buildings comprised of wide, flat floors layered atop each other in the usual, slightly uneven, pancake-stack style. Only here, these structures jut out over the valleys below like polypores (bracket or shelf fungi), sometimes far enough to give the illusion of floating above a cloudbank when the frequent dense fog fills the valley below."
```